introduction.md

```diff
@@ -0,0 +1,3 @@
+Since the JSON encoding that's generated by Apex through the serialization method in the System.JSON class isn't identical to the standard JSON encoding in some cases, the System.JSONGenerator class is provided to enable the generation of standard JSON-encoded content.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.